client/src/resources/lang/en_us.json

```diff
@@ -603,220 +603,220 @@
-    "wiki.article.role.consort.attributes":"- You can not be roleblocked\n- You can choose not to roleblock them, and only send them messages, but you will always send at least one message",
-    "wiki.article.role.consort.extra":"- You can't target other mafia members\n- When you roleblock someone, their visits will be cleared",
+    "wiki.article.role.jailor.guide": "During the day, select one player to jail each night. You can talk to your prisoner during the night, and they will not be told who you are. You should interrogate your prisoner to find out what their role is, and if you think they are evil, you can choose to execute them. You may execute up to three times, but if you execute a townie, you can not jail anyone anymore.",
+    "wiki.article.role.jailor.abilities":"* Select one player to jail during the day\n* Select your prisoner to execute them",
+    "wiki.article.role.jailor.attributes":"* You may talk with your prisoner at night\n* If you execute a townie, you forfeit the ability to jail prisoners, and lose all execution uses\n* Players in jail are immune to most abilities while in jail\n* Jailed players can't target or use their abilities\n* Other players are told when the player they visit is jailed",
+    "wiki.article.role.jailor.extra":"* The Jailor can not execute on night one\n* If you execute a townie, or run out of executions, you can not target anymore\n* You will invisibly roleblock the player you put in jail\n* Players in jail wont see your name on your messages, but the messages you send appear to be from the jailor\n* If you jail a cult or mafia member, their teammates know that that player is in jail",
+
+    "wiki.article.role.mayor.guide": "You can choose to reveal during the day to make your vote count as three votes. When you reveal, everyone is told you are the mayor.",
+    "wiki.article.role.mayor.abilities":"* Reveal during the day to triple your voting power, showing everyone that you are the Mayor",
+    "wiki.article.role.mayor.attributes":"* You get 3 votes during both voting and judgement after you reveal",
+    "wiki.article.role.mayor.extra": "* If you are converted to a different role, players will still see that you are the mayor\n* Revealing does not increase the number of votes required to put someone on trial",
+
+    "wiki.article.role.transporter.guide": "You can select two players at night. If somebody else tries to visit the first player, they will instead visit the second player, and vice versa. In general, you want to transport innocent players with suspicious ones to divert a potential attack onto the suspicious player.",
+    "wiki.article.role.transporter.abilities":"* Select two players to transport them",
+    "wiki.article.role.transporter.attributes":"* All players who visit the first player instead visit the second player\n* All visits to the second player will instead be to the first player\n* The players you visit get a message saying that they were transported",
```

Comment:
You changed the wording for one of them but not the other one, they should have the same wording in transporter attributes.

Also it is all visits to one player will be switched to to the other.

If a witch possesses the first transported player and forces them to target the second transported player. It will be the other way around. The "visits to the first player" part matters